desktop/ui/src/main/java/org/datacleaner/user/MutableReferenceDataCatalog.java

```diff
@@ -46,8 +46,11 @@
     private static final long serialVersionUID = 1L;
 
     private final List<DictionaryChangeListener> _dictionaryListeners = new ArrayList<>();
+    private final List<ReferenceDataChangeListener<Dictionary>> _dictionaryV2Listeners = new ArrayList<>();
```

Comment:
I really don't like the V2 in the field name, maybe call it _dictionaryChangeListeners instead. The same goes for the two new fields right below.